Boo/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
+get_filename_component(EXEC_NAME "${CMAKE_CURRENT_SOURCE_DIR}" NAME)
```

Comment:
The example designed for demonstration of `CONFIG` mode usage of `find_package` command. Nothing to do with the personal naming conventions you want to show.